### ISSUE TITLE
Stopping new reviews on the original site

### ISSUE THREAD

**Issue by kimadagem** [turkopticon/turkopticon-rails-app#37]
I've been in touch with Lilly about this via email; she suggested I post here to open the discussion up to more people.

I've been using both sites to make sure I get a "big picture" (history and reviews of current hit, if there are any) and I've noticed people are still posting new reviews on the original site. At first I thought it might be due to problems with the TO2 script, which wasn't working for me in Firefox. I just tried a new install and it seems fine now but I'm not sure if that's the case for everyone else. Or maybe they're using a mobile device with no script capability, or they're just staying with what's familiar. In any case, this is not going to help the new site grow and be more useful to everyone. I'm beginning to think it might be a good idea to stop accepting new reviews on the original site at some point.

I have some other thoughts:
- It's been only a couple of weeks, and the new site is still in beta. Does it have to be out of beta before people should be prevented from posting reviews on the original site? I have no experience with web development so I don't know how that would work.
- What would be involved in preventing new reviews? A script edit would help but people don't have to use the script to post reviews. Also - If people post a review right after they do a hit and then update it after the hit is paid/rejected they might need to have that ability even after new reviews are stopped.
- I'm wondering if it might also be a good idea to post about this on the forums and Reddit. At the very least it might be helpful to let everyone know in advance that the original site will eventually be closed to new reviews. If people are staying with the original site because of a problem with the new site that they haven't said anything about, this would be a good way to get that out in the open and resolved. And if they're just staying with what's familiar this might encourage them to make the change.
- If it's feasible from a coding standpoint, I think eventually it would be a good idea to just establish a "closure date," after which no new reviews can be posted on the original site - emphasizing that that site will still be available (as a kind of "read only") so people won't worry that it will disappear completely.

These are just my ideas and (due to my lack of coding skills) may not be doable.; if there are better ways to go about it please add your ideas. I just think that if the new site is going to grow it makes sense to eventually stop people from adding to the original site.

**Comment by zihel** [comment]
Yeah, ideally we'd like to be out of beta before freezing the legacy database. So what needs to be happen during beta? Generally speaking I need to do the following four things (in addition to dealing with bugs as they're discovered):

* finish implementing hotlisted features
* finish implementing a few more moderation tools for moderators
* polish the layout
* run load/performance analysis

The performance analysis is important to identify minor issues now which may or may not become larger issues when the site takes over legacy as the default TO. It lets me know what I need to be paying attention to. Anything statistically significant, however, needs to be addressed right away because that will certainly become a major problem post-transition.

**Comment by toutils** [comment]
I think one of the big things slowing people down from switching is script integration. How is this going to integrate into something like hitscraper for example? Is it going to hit both api's and integrate both sets of data somehow? That's a problem for the script developers, not the new site, but it's hard to replace 10 years worth of data, especially when some of these requesters pop up that haven't posted a hit in years, and this new review format is completely different. 

Some people are also flat out not going to want to switch for whatever reason, I don't see a way to avoid that, but I imagine forcing people to the new site is going to cause irritation, especially this quickly. I would definitely post it on reddit and the forums first to see what kind of response that's going to generate. 

I'd also like to suggest a forum of some kind for TO. turkopticon-discuss requires commenting, and that's only given out by moderators on an individual basis, I don't think there's a lot of outlets for people to talk about turkopticon specifically, aside from the forums or reddit.

**Comment by CM3Turk** [comment]
Some people may be unable to login to the beta site. Since the Beta opened for TO 2.0, I have been unable to login to the beta site. It continues to say that my username/password is invalid. So I have been unable to try the beta out, or I would be posting reviews on there everyday. I have sent an email to TO and posted the issue here so hopefully the developers can fix this issue for me and others who may be experiencing the same issue. Since the site is in "beta", I do not think it would be wise to stop reviews from being posted on the old site in case a problem occurs with the beta and people are unable to post, login or experience other issues and are then unable to return the old site to post reviews or read new reviews. However, I do believe people who are able to fully use the beta should become familiar with it and use it often to help out the developers and the community to make a better TO site. I believe the inability to use the scripts on the beta site is a main factor in why people are still using the old site. I always access TO by clicking on the access link on my script, which takes me to the old site.

**Comment by toddwc** [comment]
Is there anything new on this? Checking both sites for reviews is getting old. I see so many new reviews on TO1 and I wonder why they are not yet instead posting exclusively on TO2.

**Comment by CM3Turk** [comment]
I have noticed that as well. I had an issue logging into the legacy site earlier on but since that issue was fixed, I have been posting only to the TO2 site. Is there a  TO1 script that works with the new MTurk worker site? I only have one script that works on the new worker site but only works with the TO2 site. Since some people are only posting to the TO1 site, I have to mainly work on the old worker site because my scripts for both the TO1 and TO2 site work on it. This is definitely affecting how much I work on the new worker site. I began porting my reviews from the legacy site to the TO2 site hoping this will help.

**Comment by zihel** [comment]
@toddwc The main reason we aren't moving forward at the moment is because of issue #59. If the server is having issues keeping up with traffic during peak hours now (when only a fraction of the user base is using TO2), then it will certainly struggle when we switch everyone over. This issue needs to be resolved before we can move forward.
